packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -1119,6 +1119,21 @@ export class ProjectLifecycleManager {
     return metaState
   }
 
+  setConfigFilePath (lang: 'ts' | 'js') {
+    switch (lang) {
+      case 'ts':
+        this._configFilePath = this._pathToFile('cypress.config.ts')
+        break
+
+      case 'js':
+        this._configFilePath = this._pathToFile('cypress.config.js')
+        break
+
+      default:
+        throw new Error('Unreachable')
```

Comment:
What does this error mean if we were to hit it?

Reply:
we should not hit it, but updated it to be safe cfd16b6